src/components/Readback/readback.tsx

```diff
@@ -9,26 +9,23 @@ import { VType } from "../../vtypes/vtypes";
 import { Alarm, alarmOf, AlarmSeverity } from "../../vtypes/alarm";
 import { vtypeToString } from "../../vtypes/utils";
 
-export const Readback = (props: {
+export interface ReadbackProps {
   connected: boolean;
   value?: VType;
   precision?: number;
-  style?: {};
-}): JSX.Element => {
-  let {
-    connected,
-    value,
-    precision = undefined,
-    style = { backgroundColor: "#383838", color: "#00bb00" }
-  } = props;
+  style?: object;
+}
+
+export const Readback = (props: ReadbackProps): JSX.Element => {
+  let { connected, value, precision = undefined, style } = props;
   const alarm = alarmOf(value);
   let displayedValue;
   if (!value) {
     displayedValue = "Waiting for value";
   } else {
     displayedValue = vtypeToString(value, precision);
   }
-
+  style = { backgroundColor: "#383838", color: "#00bb00", ...props.style };
   // Change text color depending on connection state or alarm
   if (!connected) {
     style = {
```

Comment:
We can pull the below alarm sensitivity into classes in the module like `.Readback.Major` or something.